File: bukkit/libs/toollib/src/main/java/dialight/toollib/ToolMessages.java
package dialight.toollib;

import dialight.misc.Colorizer;

public class ToolMessages {

    public static final String pluginPrefix = Colorizer.apply("|go|Инструмент|gr|: ");

    public static String notFound(String id) {
        return pluginPrefix + Colorizer.apply("|r|Инструмент с ID «$id» не зарегистрирован");
    }

}

File: sponge/libs/toollib/src/main/kotlin/dialight/toollib/ToolMessages.kt
package dialight.toollib

import jekarus.colorizer.Text_colorized
import org.spongepowered.api.text.Text

object ToolMessages {

    var pluginPrefix = Text_colorized("|go|Инструмент|gr|: ")

    fun notFound(id: String): Text = Text.of(pluginPrefix, Text_colorized("|r|Инструмент с ID «$id» не зарегистрирован"))


}